src/plugins/score-plugin-midi/Midi/MidiDrop.hpp

```diff
@@ -18,11 +18,101 @@ class DropHandler final : public Process::ProcessDropHandler
       const score::DocumentContext& ctx) const noexcept override;
 };
 
+struct PitchbendData{
```

Comment:
MAybe all of these could go into a more general Midi.hpp ... or, I wonder if it wouldn't be best in libremidi, and then try to consume those directly ? not sure...

Reply:
Thanks for the suggestion, I'll start moving it soon then, please let me know what makes most sense.

I was going with MidiTrackEvent as type name (which is the core object along MidiNote which has "pointers" to optional? pair), and considering a "domain" folder:

/domain/MidiNote.hpp moving the current one
/domain/MidiTrackEvent.hpp the detailed header
/domain/Midi.hpp a logical header, including several others, for convenience in other modules.

Do you like this?